src/wireviz/wireviz.py

```diff
@@ -86,6 +86,8 @@ def parse(
         raise Exception("No output formats or return types specified")
 
     yaml_data, yaml_file = _get_yaml_data_and_path(inp)
+    if not isinstance(yaml_data, dict):
+        raise TypeError(f"Expected a dict/map as top-level YAML input, but got: {type(yaml_data)}")
```

Comment:
What does map mean in this context?

Reply:
Some YAML descriptions use "map", others call it "dictionary", and "hash" is also used. In the Python world, I guess "dict" is more common, so I have now changed it to a simple "dict". See e.g. https://www.yaml.info/learn/index.html